include/runtime_symbol_lookup.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "ddprof_defs.hpp"
+#include "symbol_table.hpp"
+
+#include <map>
+#include <unordered_map>
+
+namespace ddprof {
+
+class RumtimeSymbolVal {
```

Comment:
`Runtime` instead of `Rumtime`.
Not sure it's the best name, why not reuse `DwflSymbolVal` ?

Reply:
I will propose `SymbolSpan` if that is ok.